src/lib/types/model.ts

```diff
@@ -198,7 +198,7 @@ export interface IEnvironment {
     projectCount?: number;
     apiTokenCount?: number;
     enabledToggleCount?: number;
-    requiredApprovals?: number;
+    requiredApprovals?: number | null;
```

Comment:
we have to distinguish null and undefined. undefined means skip this field in update and keep the previous value, null means reset this field to null in update